symfony/framework-bundle/3.3/config/packages/framework.yaml

```diff
@@ -4,10 +4,9 @@ framework:
     #csrf_protection: ~
     #http_method_override: true
 
-    # uncomment this entire section to enable sessions
-    #session:
-    #    # With this config, PHP's native session handling is used
-    #    handler_id: ~
+    session:
```

Comment:
I'd add a comment here to explain that it doesn't harm their application's performance.

Reply:
And also perhaps a note about how they could remove this to explicitly disable sessions?

Reply:
Any suggestions?

Reply:
```yaml
# Enable session support. Note that the session will be started ONLY if you read or write from it.
# Remove or comment this section to disable the session support.
session:
```

Reply:
👍 I rudely made a few minor tweaks to your suggestion - hope you don't mind @sroze ;)

Reply:
That’s perfect. I’ll survive it 😜

Reply:
updated, thanks